Non-backwards-compatible version released to crates.io with only minor version bump

Thanks so much for this crate and ratelimit_futures, they've been super useful to me.

The recent 4.1.0 release of ratelimit_meter seems to be backward-incompatible with 4.0.x, since ratelimit_futures no longer builds:

```
error[E0107]: wrong number of type arguments: expected 1, found 0
  --> src/lib.rs:65:41
   |
65 |     <A as Algorithm>::NegativeDecision: NonConformance,
   |                                         ^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
  --> src/lib.rs:74:41
   |
74 |     <A as Algorithm>::NegativeDecision: NonConformance,
   |                                         ^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> src/lib.rs:101:41
    |
101 |     <A as Algorithm>::NegativeDecision: NonConformance,
    |                                         ^^^^^^^^^^^^^^ expected 1 type argument
```

... but only the minor version was bumped, so the ratelimit_futures crate (and anything else depending on ratelimit_meter using a normal "caret" dependency) automatically pulls the newer version and breaks.

--- a/src/algorithms/mod.rs
+++ b/src/algorithms/mod.rs
@@ -23,7 +23,7 @@ pub type DefaultAlgorithm = LeakyBucket;
 ///
 /// Since this does not account for effects like thundering herds,
 /// users should always add random jitter to the times given.
-pub trait NonConformance<P: instant::Relative> {
+pub trait NonConformance<P: instant::Relative = instant::TimeSource> {
     /// Returns the earliest time at which a decision could be
     /// conforming (excluding conforming decisions made by the Decider
     /// that are made in the meantime).

